site/topics/logic/boolean-logic.rst

```diff
@@ -61,35 +61,35 @@ Boolean Operators and Operands
     :align: center
     :target: https://en.wikipedia.org/wiki/Boolean_algebra#Boolean_operations
 
-    Visual representations of the basic boolean operators with Venn diagrams. The images represent **and**, **or**, and
-    **not** respectively. 
+    Visual representations of the basic boolean operators with Venn diagrams. The images represent AND, OR, and NOT
+    respectively.
 
 
 * There are additional boolean operators that can be made up from the three basic operators
 * Three of these are commonly used within the context of computer architecture, thus they will be presented here
 
-    * **Exclusive Or** (**XOR**)
+    * **Exclusive OR** (**XOR**)
```

Comment:
I left "Exclusive" not capitalized, but changed NOT AND/NOT OR. 

Motivation is. 
NOT AND is literally the NOT and AND operators
"Exclusive" is not an operator

In other words, there is method to the madness